src/main/webapp/app/entities/file-upload-exercise/file-upload-exercise-update.component.html

```diff
@@ -95,7 +95,25 @@ <h2 *ngIf="!fileUploadExercise.id" id="jhi-file-upload-exercise-heading-create"
                         placement="top"
                         ngbTooltip="{{ 'artemisApp.fileUploadExercise.filePatternInfo' | translate }}"
                     ></fa-icon>
-                    <input required minlength="2" type="text" class="form-control" name="filePattern" id="field_filePattern" [(ngModel)]="fileUploadExercise.filePattern" />
+                    <form [formGroup]="fileUploadForm">
```

Comment:
Please don't put form elements in other form elements. We already have a form around the whole edit page.